tests/unit/all.ts

```diff
@@ -19,3 +19,6 @@ import './string';
 import './UrlSearchParams';
 import './WeakMap';
 import './DateObject';
+import './async/iteration';
```

Comment:
Not sure how this happened, but these async modules are already listed in all.ts (at the top, in alphabetical order).  Map should also be added in case-insensitive alpha order here.